xendit-android/src/main/java/com/xendit/Models/CardInfo.java

```diff
@@ -0,0 +1,40 @@
+package com.xendit.Models;
+
+import com.google.gson.annotations.SerializedName;
+
+public class CardInfo {
+    @SerializedName("bank")
+    private String bank;
+
+    @SerializedName("country_code")
```

Comment:
change this to `country`.

review from API team https://github.com/xendit/apireference/pull/362